composer/trainer/dist_strategy.py

```diff
@@ -508,6 +510,9 @@ def _auto_wrap_policy_old(module: torch.nn.Module, recurse: bool, unwrapped_para
                 **kwargs,
             )
 
+            fsdp_obj._exec_order_data._forward_prefetch_limit = fsdp_config['forward_prefetch_limit']
```

Comment:
Should probably check for the existence of these attrs since these are private...

Reply:
I would prefer to leave as is since I have verified on torch versions so far, and I would rather have test fail while we upgrade to new torch so it flags as requiring change

Reply:
These can change in a patch release though, which will instantly break everyone's training jobs.

Reply:
Should also probably warn, with a note saying to open a composer issue